### PR TITLE
fixed type of save button

### PR DESCRIPTION
**Pull request by edewit**
fixing: #632

### CODE REVIEW

File: src/clients/advanced/SaveReset.tsx [hunk 1]
```diff
@@ -9,14 +9,14 @@ type SaveResetProps = ActionGroupProps & {
 };
 
 export const SaveReset = ({ name, save, reset, ...rest }: SaveResetProps) => {
-  const { t } = useTranslation();
+  const { t } = useTranslation("common");
   return (
     <ActionGroup {...rest}>
-      <Button data-testid={name + "Save"} variant="tertiary" onClick={save}>
```
Comment (mfrances17) [inline src/clients/advanced/SaveReset.tsx:15]: Was there a special request to use the tertiary variant here? I would think we'd want this to be primary (the default).

Comment (edewit) [reply src/clients/advanced/SaveReset.tsx:15]: right that's why this PR changes it to be "default"

Comment (mfrances17) [reply src/clients/advanced/SaveReset.tsx:15]: @edewit DOH! sorry, i read the diff wrong... all set, thanks

**Review by mfrances17 — APPROVED** [review]
LGTM

**Pull request merged**